torchrl/modules/tensordict_module/common.py

```diff
@@ -80,8 +80,8 @@ def _forward_hook_safe_action(module, tensordict_in, tensordict_out):
                 )
 
 
-class TensorDictModule(_TensorDictModule):
-    """A TensorDictModule, is a python wrapper around a :obj:`nn.Module` that reads and writes to a TensorDict.
+class SafeModule(TensorDictModule):
+    """A SafeModule, is a python wrapper around a :obj:`nn.Module` that reads and writes to a TensorDict.
```

Comment:
`An :obj:``SafeModule`` is a :obj:``tensordict.nn.TensorDictModule`` subclass that accepts a :obj:``TensorSpec`` as argument to control the output domain.`